entry_types/scrolled/package/src/frontend/registerTemplateWidgetType.js

```diff
@@ -0,0 +1,4 @@
+export default function(typeName, callback) {
+  let element = document.getElementById('template-widget-container');
+  callback(element);
+}
```

Comment:
In Pageflow Paged, one of the main responsibilities of `widgetTypes.register` is to ensure that only the `enhance` method of those widget types are called that are actually present in the document. If we introduce a Scrolled counterpart, we should make sure the same is true.

Alternatively, we can remove the function completely and make sure our existing widgets check themselves whether there rendered output is present.

Reply:
As discussed, for now we go with the alternative you describe in the second paragraph. Up until now, I think there aren't any more widgets for which we'd like scrolled compatibility.